tracy-client/src/lib.rs

```diff
@@ -44,6 +44,9 @@ mod plot;
 mod span;
 mod state;
 
+#[cfg(feature = "demangle")]
+pub mod demangle;
```

Comment:
It's already exported through macro_export, do you want to do this regardless?

Reply:
Yes. Among other reasons it makes `use crate::register_demangler` work within the crate without surprises...

Reply:
```rust
error[E0432]: unresolved import `crate::demangle::register_demangler`
  --> tracy-client/src/lib.rs:31:9
   |
31 | pub use crate::demangle::register_demangler;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `register_demangler` in `demangle`
   |
   = note: this could be because a macro annotated with `#[macro_export]` will be exported at the root of the crate instead of the module where it is defined
```